src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.ts

```diff
@@ -75,6 +84,15 @@ export class ExamNavigationBarComponent implements OnInit {
         this.setExerciseButtonStatus(this.exerciseIndex);
     }
 
+    /**
+     * Auxiliary method to call changeExerciseByIndex based on the unique id of the exercise
+     * @param exerciseId the unique identifier of an exercise that stays the same regardless of student exam ordering
+     */
+    changeExerciseById(exerciseId: number) {
+        const foundIndex = this.exercises.findIndex((ex) => ex.id === exerciseId);
```

Comment:
you should use a better and more descriptive name instead of `ex` to adhere to the style guidelines. Using `exercise` instead would be better.

Reply:
Good catch. Thank you (I was pushing the client style issues fix the moment you typed your comment 😀)